model/group_request.go

```diff
@@ -105,6 +107,12 @@ func (p *PatchGroupRequest) Apply(group *Group) bool {
 		}
 	}
 
+	// This special value allows us to bump the group sequence number even when
+	// the patch contains no group modifications.
+	if p.ForceSequenceUpdate {
```

Comment:
0/5 you could simplify this to just `p.ForceSequnceUpdate = applied` couldn't you?

Reply:
`applied` is actually the important var that is influenced by `p.ForceSequenceUpdate` and `p.ForceSequenceUpdate` being false doesn't mean `applied` should be false. I think this is about as clean as you can make it, but thanks for the input!